css/interface.css

```diff
@@ -664,6 +664,9 @@ td.align-baseline {
 
 .ui-autocomplete {
   z-index: 9999; /* goes above modal */
+  max-height: 200px;
```

Comment:
Why 200px?

Reply:
I decided to use 200px because there is actually about 200px from input to the bottom of the modal:
Row with input min height - 88, next row - 68, footer - 71, `hr` margins - 30px
Assuming that dropdown opens exactly under input (decrease by input and `hr` bottom margin): 88 + 68 + 71 + 30 - 10 - 48 = 199
![image](https://user-images.githubusercontent.com/74735207/112982609-5775e280-9165-11eb-8655-cf52c057147a.png)
![image](https://user-images.githubusercontent.com/74735207/112982071-b555fa80-9164-11eb-82bd-e623040be860.png)

Reply:
👍🏻 

Can you add a comment `/* Prevent dropdown from being clipped within the overlay frame */` please to explain this?

Reply:
Sure, I added the comment.